comp/core/autodiscovery/autodiscoveryimpl/configmgr.go

```diff
@@ -51,6 +51,13 @@ type configManager interface {
 
 	// getActiveServices returns the currently active services
 	getActiveServices() map[string]listeners.Service
+
+	// configsForSecretRefresh returns the raw config paired with the resolved
+	// configs currently scheduled for the given origin digest.
+	configsForSecretRefresh(origin string) (integration.Config, []integration.Config, bool)
```

Comment:
I think it might be better to have a function name that describe what the function does instead of where it's used.

Maybe `getConfigByDigest(digest string) (integration.Config, []integration.Config, bool)` ?